src/WindowsUtils/CreateProcessTest.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/ExecutablePath.h"
+#include "Test/Path.h"
+#include "TestUtils/TestUtils.h"
+#include "WindowsUtils/CreateProcess.h"
+#include "WindowsUtils/ProcessList.h"
+
+namespace {
+std::filesystem::path GetTestExecutablePath() {
+  static auto path = orbit_base::GetExecutableDir() / "FakeCliProgram.exe";
```

Comment:
Should that FakeCliProgram.exe also be part of the PR?

Reply:
It's already compiled and available actually.